torch/testing/_asserts.py

```diff
@@ -102,6 +102,69 @@ def wrapper(actual: Tensor, expected: Tensor, **kwargs: Any) -> Optional[Excepti
     return wrapper
 
 
+def _check_sparse_coo_members_individually(
+    check_tensors: Callable[..., Optional[Exception]]
+) -> Callable[..., Optional[Exception]]:
+    """Decorates strided tensor check functions to individually handle sparse COO members.
+
+    If the inputs are not sparse COO, this decorator is a no-op.
+
+    Args:
+        check_tensors (Callable[[Tensor, Tensor], Optional[Exception]]): Tensor check function for strided tensors.
+    """
+
+    @functools.wraps(check_tensors)
+    def wrapper(actual: Tensor, expected: Tensor, **kwargs: Any) -> Optional[Exception]:
+        if not actual.is_sparse:
+            return check_tensors(actual, expected, **kwargs)
+
+        exc = check_tensors(actual._indices(), expected._indices(), **kwargs)
```

Comment:
Currently we are using the same `**kwargs` for checking the indices and values. That means this might include an `rtol` and `atol` `> 0`. It would be better to check indices for equality, correct? This also applies to `crow_indices` and `col_indices` of sparse CSR.

Reply:
Correct, all indices should be checked for equality.

Reply:
Use:
```
exc = check_tensors(actual._indices(), expected._indices(), **dict(kwargs, rtol=0.0, atol=0.0))
```